### PR TITLE
Updates on bootstrap library to the latest stable version and the README file

### PR DESCRIPTION
**Pull request by ElinorW**

### CODE REVIEW

**Review by titusgicheru — COMMENTED** [review]
Please change "Copyright (c) 2018.." to 2019 on the line below otherwise it looks good.

[https://github.com/microsoftgraph/aspnet-todo-rest-sample/pull/7/commits/9cebddedebd5f933323800f16da33953e72e4873#diff-04c6e90faac2675aa89e2176d2eec7d8R86](https://github.com/microsoftgraph/aspnet-todo-rest-sample/pull/7/commits/9cebddedebd5f933323800f16da33953e72e4873#diff-04c6e90faac2675aa89e2176d2eec7d8R86)

**Review by titusgicheru — APPROVED** [review]

**Review by irvinesunday — CHANGES_REQUESTED** [review]
Please use section highlights on the screenshots for step 5 and 7 as you've done with the screenshots for steps 1 and 2 (and maybe even indicate sequence step numbers alongside them).

**Comment by ElinorW** [comment]
I have made the requested changes

File: README.md [hunk 1]
```diff
@@ -11,28 +11,36 @@ This sample requires the following:
...
+5. Select **Certificates & secrets** under **Manage**. Select the **New client secret** button. Enter a value in **Description**, select any option for **Expires** and choose **Add**.  
+![Certificates & secrets](images/ClientSecretRegistration.PNG)
  
-6. Open the sample solution in Visual Studio and then open the **Constants.cs** file. Change the **Tenant** string to the **Directory (tenant) ID** value you copied earlier. Similarly, change the **ClientIdForAppAuthn** string to the **Application (client) ID** value and change the **ClientSecret** string to the client secret value. 
+6. Copy the client secret value before leaving the page. You will need it in the next step.  
+ ![Certificates & secrets](images/ClientSecret.PNG)
  
-7. Return to the Azure Active Directory management center. Select **API permissions** and then select **Add a permission**. On the panel that appears, choose **Microsoft Graph** and then choose **Application permissions**. 
+7. Select **API permissions** and then select **Add a permission**. On the panel that appears, choose **Microsoft Graph** and then choose **Application permissions**.
```
Comment (irvinesunday) [inline README.md]: There is no need for this step as scopes are defined explicitly in the config file. See https://github.com/microsoftgraph/aspnet-todo-rest-sample/blob/249e3f6b898ce7bcd3accc5086a9098317f943c5/Microsoft-Graph-ExcelRest-ToDo/Web.config#L14

Comment (ElinorW) [reply README.md]: Okay, I have deleted the steps.

**Review by irvinesunday — APPROVED** [review]

**Pull request merged**